docs/model_library/water_treatment/index.rst

```diff
@@ -0,0 +1,780 @@
+
+    F_{in} = F_{brine}^{(I)} + F_{vapor}^{(I)}
+
+    F_{in}^{(i+1)} = F_{brine}^{(i)} + F_{vapor}^{(i)} \quad \forall i < I
+
+Flow matching between super heated vapor and flow of vapor from the evaporator:
+
+.. math:: 
+
+    F_{spv} = F_{vapor}^{(I)}
+
+Mass balance in the evaporators:
+
+.. math:: 
+
+    F_{in}Q_{in} = F_{brine}^{(I)}Q_{brine}^{(I)}
+
+    F_{brine}^{(i+1)}Q_{brine}^{(i+1)}=F_{brine}^{(i)}Q_{brine}^{(i)} \quad \forall i < I
+
+Calculating mass fraction of salt from salt salinity:
+
+.. math:: 
+
+    X_{s}^{in} = 0.001 Q_{in}
+
+    X_s^{(i)} = 0.001 Q_{brine}^{(i)} \quad \forall i \in \{0,...,I\}
+
+Energy balance in the evaporator:
+
+.. math:: 
+
+    E^{(I)} + F_{in}H_{in} = F_{brine}^{(I)}H_{brine}^{(I)} + F_{vapor}^{(I)}H_{vapor}^{(I)}
+
+    E^{(i)} + F_{brine}^{(i+1)}H_{brine}^{(i+1)} = F_{brine}^{(i)}H_{brine}^{(i)} + F_{vapor}^{(i)}H_{vapor}^{(i)} \quad \forall i < I
+
+    E^{(1)} =  F_{spv}C_p^{vapor}(T_{spv} - T_{cond}^{(1)}) + F_{spv}(H_{cond}^{vap(1)} - H_{cond}^{(1)})
+
+    E^{(i)} = F_{vapor}^{(i-1)} \lambda^{(i)} \quad \forall i \in \{2,...,I\}
+
+Thermodynamic Relations
+
+Relating pressures to temperatures using the Antoine equation: 
+
+.. math:: 
+
+    log(P_{vapor}^{(i)}) = a + \frac{b}{T_{ideal}^{(i)} + c} \quad \forall i \in \{1,.., I\}
+
+    log(P_{spv}^{(1)}) = a + \frac{b}{T_{cond}^{(1)} + c}
+
+    log(P_{sv}^{(i)}) = a + \frac{b}{T_{sv}^{(i)} + c} \quad \forall i \in \{2,..., I\}
+
+Calculating elevation in boiling point due to TDS in the feed water:
+
+.. math:: 
+
+    BPE^{(i)} = 0.1581 + 2.769 X_{s}^{(i)} - 0.002676 T_{ideal}^{(i)}+ 41.78 X_{s}^{(i)2} + 0.134 X_{s}^{(i)}T_{ideal}^{(i)}
+
+Calculating temperature of brine from BPE and ideal temperature in the evaporator:
+
+.. math:: 
+
+    T_{brine}^{(i)} = T_{ideal}^{(i)} + BPE^{(i)}
+
+Estimating the enthalpies:
+
+.. math:: 
+
+    H_{in} = -15940 + 8787X_{s}^{in} + 3.557 T_{in}
+
+    H_{brine}^{(i)} = -15940 + 8787X_{s}^{(i)} + 3.557 T_{brine}^{(i)}\quad \forall i \in \{1,..., I\}
+
+    H_{vapor}^{(i)} = -13470 + 1.84 T_{brine}^{(i)}\quad \forall i \in \{1,..., I\}
+
+    H_{cond}^{(i)vap} = -13470 + 1.84T_{cond}^{(i)}\quad \forall i \in \{1,..., I\}
+
+    H_{cond}^{(i)} = -15940 + 3.557T_{cond}^{(i)}\quad \forall i \in \{1,..., I\}
+
+Calculating LMTD:
+
+.. math:: 
+
+    \theta_1^{(i)} = T_{spv} - T_{brine}^{(i)} \quad \text{for } i = 1
+
+    \theta_1^{(i)} = T_{sv}^{(i)} - T_{brine}^{(i)} \quad \forall i > 1
+
+    \theta_2^{(i)} = T_{cond}^{(i)} - T_{brine}^{(i+1)} \quad \text{for } i = 1
+
+    \theta_2^{(i)} = T_{sv}^{(i)} - T_{brine}^{(i+1)} \quad \forall i \in \{2,.., I-1\}
+
+    \theta_2^{(i)} = T_{sv}^{(i)} - T_{in} \quad \text{for } i = I
+
+    LMTD^{(i)} = (0.5\theta_1^{(i)}\theta_2^{(i)}(\theta_1^{(i)}+\theta_2^{(i)}))^{1/3}
+
+Heat transfer coefficient for evaporator:
+
+.. math:: 
+
+    U_{evap}^{i} = 0.001(1939.4 + 1.40562T_{brine}^{(i)} - 0.002T_{brine}^{(i)2}+ 0.0023T_{brine}^{(i)3})
+
+Design Equations:
```

Comment:
```suggestion
**Design Equations**
```
Same comment here - I think this should be bold. And drop the colon to be consistent.